scripts/emerge/stage_prs_anvil_deliverable_data.py

```diff
@@ -0,0 +1,193 @@
+# imports and environment variables
+import argparse
+import json
+import pandas as pd
+
+from firecloud import api as fapi
+from google.cloud import storage as gcs
+
+
+def upload_data_table(tsv, workspace, namespace):
+    """Upload tsv load file to workspace to create/add to existing data table."""
+
+    upload_response = fapi.upload_entities_tsv(namespace, workspace, tsv, model='flexible')
+    status_code = upload_response.status_code
+
+    if status_code != 200:  # api call fails
+        print(f"Uploading entity failed: {upload_response.text}")
+        return
```

Comment:
again here, do you want the script to continue if one of the uploads fails? i'd think you'd want the WDL to fail, but not sure